cmd/cadvisor.go

```diff
@@ -192,6 +204,10 @@ func main() {
 	// Register Prometheus collector to gather information about containers, Go runtime, processes, and machine
 	cadvisorhttp.RegisterPrometheusHandler(mux, resourceManager, *prometheusEndpoint, containerLabelFunc, includedMetrics)
 
+	muxHandle := nethttp.Middleware(opentracing.GlobalTracer(), mux, nethttp.OperationNameFunc(func(r *http.Request) string {
```

Comment:
What would the operation name be if we didn't rename it here?

Reply:
nit: we should be able to assign this back to mux so we don't have to modify variable names below.

Reply:
1. the default operation name is `"HTTP " + http.Method` ex: `HTTP GET`. That is a bit useless.
2. the mux variable is alllocated as `ServerMux` and a reassignement is not possible. Because the `nethttp.Middleware` returns a `http.Handle`